source/_components/sensor.dominionenergy.markdown

```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+title: Dominion Energy Sensor
+description: "Instructions on how to set up Dominion Energy within Home Assistant."
+date: 2018-03-21 08:00
+sidebar: false
+comments: false
+sharing: true
+footer: true
+logo: dominionenergy.jpg
+ha_category: Sensor
+ha_release: 0.55.7
```

Comment:
Should be updated once this platform is added. Probably 0.67.0

Reply:
Took care of this.